scripts/check_command_docs

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
+# Copyright (c) 2021 elParaguayo
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+import argparse
+import importlib
+import importlib.util
+import inspect
+import os
+import sys
+from pathlib import Path
+
+this_dir = os.path.dirname(__file__)
+base_dir = os.path.abspath(os.path.join(this_dir, ".."))
+sys.path.insert(0, base_dir)
+
+from libqtile.command.base import CommandObject  # noqa: E402
+
+
+def folder(value):
+    if not os.path.isdir(value):
+        raise argparse.ArgumentTypeError("Invalid root folder.")
+    return value
+
+
+class CommandChecker:
+    def __init__(self, path):
+        self.path = path
+        self._find_files()
+        self.errors = 0
+
+    def _find_files(self):
+        modules = {}
+        all_files = [f.as_posix() for f in Path(self.path).rglob("*.py")]
+        for f in all_files:
+            path = f.split("/")
+            try:
+                qtile = f.index("libqtile")
+            except ValueError:
+                continue
+
+            modules[f] = ".".join(path[qtile:])[:-3]
+
+        self.modules = modules
+
+    def check(self):
+        for path, modulename in self.modules.items():
```

Comment:
It isn't immediately obvious what these methods do without really reading the code - perhaps add some docstrings so that in a year's time someone reading this understands what this is for?